website/frontend/templates/docs/faq.html

```diff
@@ -182,6 +182,24 @@ <h3 id="fedora-acoustid">
 
       <h1 id="configuration">Configuration</h1>
 
+      <h3 id="config-location">
+      Where is the Picard configuration saved?</h3>
+
+      <p>
+        Picard saves the configuration in the file <code>Picard.ini</code>. The location of the
+        configuration depends on the operating system:
```

Comment:
s/The location of the configuration depends/Its location depends/ ?

Reply:
Yes, makes sense. Changed